verification.cs

```diff
@@ -0,0 +1,13 @@
+// .NET 5
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+var key = "";
```

Comment:
Is this the right function/method to pull the key from an environment variable?

`var key = System.Environment.getEnvironmentVariable("CHAMELEON_VERIFICATION_SECRET");` would this work for this line?

Reply:
It would be capital "G" in `GetEnvironmentVariable`

```c#
var key = System.Environment.GetEnvironmentVariable("CHAMELEON_VERIFICATION_SECRET");
```

I made the `key` open ended because there are various ways of getting it. So it will be easier to start trying the code by hard-coding it.